tests/settings/ConfigurationGatheringTest.py

```diff
@@ -362,7 +362,8 @@ def test_default_section_deprecation_warning(self):
                                  self.log_printer,
                                  arg_list=[])
 
-        self.assertIn('WARNING', cm.output[0])
+        output = '\n'.join(cm.output)
+        self.assertIn('WARNING', output)
```

Comment:
why this change?

Reply:
@Makman2 coala is giving both warnings and errors through stderr. so I needed a condition such as there is atleast one warning created. I am giving out an error now so `cm.output[0]` is no longer a warning

Reply:
I couldn't find something to match atleast one from a list in assert